### ISSUE TITLE
Adding different types of editors we can use in the linux cheatsheet

### ISSUE THREAD

**Issue by jaisinha** [Suryansh1720001/Developer-Cheatsheet#26]

**Comment by Suryansh1720001** [comment]
Hello @jaisinha 
Go for it. And go through the readme and contributing file first before sending the pull request.
Thank you

**Comment by jaisinha** [comment]
Done,,,pl take a look..and also a question..do I have any chance to get a t-shirt now...my 3/4 prs have been accepted

**Comment by jaisinha** [comment]
@Suryansh1720001 reply pl

**Comment by Suryansh1720001** [comment]
Yes, you can win the swags after you complete your 1 PR which is remaining.
